autotest/gcore/vsizip.py

```diff
@@ -927,9 +935,16 @@ def test_vsizip_byte_sozip():
 
         ds = gdal.Open(dstfilename)
         assert ds.GetRasterBand(1).Checksum() == 4672
+        del ds
```

Comment:
Is this supposed to call the destructor? It's not a great guarantee in Python that deleting the object will cause it to be destroyed immediately. Presumably there is a close method to call instead?

Reply:
indeed there's a Close() method. Just used it